### PR TITLE
docs: update usage guides and screenshot

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- Update usage guide and screenshots for
Introduction
Customizing
Importing Data

- And I also very annoyed by the absence of distance between images and the text below, so I add margin there.

### CODE REVIEW

File: docs/_static/rotkehlchen_docs.css [hunk 1]
```diff
@@ -1,6 +1,11 @@
 .wy-side-nav-search > a {
   width: 175px;
 }
+
 .red {
   color: red;
 }
+
+.rst-content section > img {
+  margin-bottom: 24px !important;
```
Comment (LefterisJP) [inline docs/_static/rotkehlchen_docs.css:10]: what does this do?

File: docs/usage_guide.rst [hunk 2]
```diff
@@ -295,6 +295,11 @@ Animations effect
 
 It is possible to reduce the animation effect in different parts of the app to improve the performance if needed. This will affect the quality of the animations but the user will get a better usage of the resources.
 
+Alias name for addresses
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, rotki will use alias name for blockchain addresses. You can choose to disable this feature. The alias name is obtained from ``ENS``, ``addressbook``, or ``blockchain account label`` and you also can change the order of priority.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:301]:
```suggestion
By default, rotki will use an alias name for blockchain addresses if one can be found. You can choose to disable this feature. The alias name is obtained from ``ENS``, ``addressbook``, or ``blockchain account label`` and you also can change the order of resolution of that name.
```

**Review by LefterisJP — COMMENTED** [review]
small comments.

Nice screenshot changes

File: docs/_static/rotkehlchen_docs.css (hunk 1 above)
Comment (lukicenturi) [reply docs/_static/rotkehlchen_docs.css:10]: this add margin after every images.

Before
image

After
image

**Review by LefterisJP — APPROVED** [review]
LGTM

**Pull request merged**